content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,357 @@ schema: olm.channel
 
+## Composite Template
+You can use a `composite template` to generate a valid FBC for multiple catalogs at a time. A `composite template` is an input schema that accepts a configuration schema. Catalog maintainers can use a `composite template` to build multiple versions of a catalog while maintaining fewer files. Operator authors can use a `composite template` to contribute their operator to multiple catalogs.
+
+### Usage
+```sh
+opm alpha render-template composite [flags] <filename>
+```
+
+| Flag                | Description                                                                            |
+| ------------------- | -------------------------------------------------------------------------------------- |
+| -h, --help          | prints help/usage information                                                          |
+| -o, --output <type> | the output format, can be `yaml` or `json`                                             |
+| -c, --composite-config <file> | File to use as the composite configuration file (default "catalog/config.yaml") |
+| --skip-tls-verify   | skip TLS certificate verification for container image registries while pulling bundles |
+| --use-http          | use plain HTTP for container image registries while pulling bundles                    |
+| --validate          | whether or not the created FBC should be validated (i.e 'opm validate') (default true) |
+| --container-tool <tool> | container tool to be used when rendering veneers (should be an equivalent replacement to docker - similar to podman) (default "docker") |
+
+## Specification
+The composite veneer consists of the following schemas:
+- `olm.composite`
+    - This schema defines what, where, and how to build a particular contribution
+- `olm.composite.catalogs`
+    - This schema defines a catalog and the constraints for building the catalog
+
+The `olm.composite.catalogs` cue schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+The `olm.composite` cue schema is:
+```
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
+	Name:        string
+	Destination: #ComponentDestination
+	Strategy:  #BuildStrategy
+}
+
+#ComponentDestination {
+	Path: string
+}
+
+#BuildStrategy: {
+	Name:   string
+	Template: #TemplateDefinition
+}
+
+#TemplateDefinition: {
+	Schema: string
+	Config: _ @go(,json.RawMessage)
+}
+```
+
+### Examples
+**Catalog Config File**
+
+The following example creates a catalog configuration that:
+- Defines the `v4.10` catalog and restricts contributions to the catalog to only being `basic` templates
```

Comment:
I have mixed feelings about referring to OCP versions in an upstream doc.  Of course we understand the references, but there may be audiences who reach this doc who won't be able to infer the significance.  Would it be better if we abstracted like [v1, v2, v3] or [one, two, three]?